synapse/storage/databases/main/receipts.py

```diff
@@ -197,7 +197,7 @@ async def get_receipts_for_user_with_orderings(
         self, user_id: str, receipt_type: str
     ) -> JsonDict:
         """
-        Fetch receipts in all rooms for a user.
+Fetch receipts for all rooms that the given user is joined to.
```

Comment:
Sorry, I screwed up the formatting.

```suggestion
        Fetch receipts for all rooms that the given user is joined to.
```